.docker/Dockerfile

```diff
@@ -1,9 +1,88 @@
-FROM jekyll/jekyll:4.0.0
+FROM ruby:2.7.1
 
-RUN apk add --no-cache --no-progress\
-    graphviz\
-    openjdk8-jre-base\
-    fontconfig\
-    ttf-dejavu
+ENV BUNDLE_SILENCE_ROOT_WARNING=1 \
+    BUNDLE_APP_CONFIG=/usr/local/bundle \
+    BUNDLE_HOME=/usr/local/bundle \
+    BUNDLE_APP_CONFIG=/usr/local/bundle \
+    BUNDLE_BIN=/usr/local/bundle/bin
+ENV GEM_BIN=/usr/gem/bin \
+    GEM_HOME=/usr/gem
+ENV JEKYLL_VAR_DIR=/var/jekyll \
+    JEKYLL_VERSION=4.0.0 \
+    JEKYLL_DATA_DIR=/srv/jekyll \
+    JEKYLL_BIN=/usr/jekyll/bin \
+    JEKYLL_ENV=development
+ENV PATH=/usr/jekyll/bin:/usr/local/bundle/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ENV VERBOSE=false
+ENV FORCE_POLLING=false
+ENV DRAFTS=false
 
-RUN gem install bundler
+RUN apt-get clean && \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5414481)